Echo/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set( PSYNC_HOME
     )
 
```

Comment:
Improvements addressed and completed here: 
https://github.com/PolySync/PolySync-CPP-Examples/pull/7/commits/81a655e1d1a9ed1b22a1b6684daddeaabfef2a02